pkg/v1/tkg/test/tkgctl/aws_cc/aws_cc_suite_test.go

```diff
@@ -71,7 +71,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 
 	By(fmt.Sprintf("Loading the e2e test configuration from %q", e2eConfigPath))
 	e2eConfig = framework.LoadE2EConfig(context.TODO(), framework.E2EConfigInput{ConfigPath: e2eConfigPath})
-	Expect(e2eConfigPath).ToNot(BeNil(), "Failed to load e2e config from %s", e2eConfigPath)
+	Expect(e2eConfig).ToNot(BeNil(), "Failed to load e2e config from %s", e2eConfigPath)
```

Comment:
why the change? unless it changes code behavior, even if it is really ugly name don't change variable names unless you have to.

Reply:
Cause it was a bug before. ```e2eConfig``` has just been created in the line above should have been tested for nilness not ```e2eConfigPath```

Reply:
I haven't changed variable names

Reply:
never mind. I see the reason.